Sources/Utility/ArgumentParser.swift

```diff
@@ -398,12 +486,14 @@ public final class ArgumentParser {
     /// Overview text of this parser.
     let overview: String
 
-    /// The parser contains one and only optional positional argument.
-    private var optionalPositionalArg = false
-
     /// If this parser is a subparser.
     private let isSubparser: Bool
 
+    /// Boolean specifying if the parser can accept further positional
+    /// arguments (false if it already has one optional or remaining
+    /// positional argument).
+    private var canAcceptPositionalArguments: Bool = true
```

Comment:
A positional argument with a `.remaining` array parsing strategy. But it's a mistake, it also concerns optional arguments. I'll have to fix it.